scripts/releaseCommon.py

```diff
@@ -117,3 +118,18 @@ def updateConanTestFile(self):
         f = open( conanTestPath, 'w' )
         for line in lines:
             f.write( line + "\n" )
+
+    def updateTravisFile(self):
+        conanParser = re.compile( r'         CONAN_REFERENCE: \"Catch\/\d+\.\d+\.\d+.*\"')
+        f = open( travisPath, 'r' )
```

Comment:
I would always suggest the ``with open(travisPath, "r") as f:`` context idiom, it makes sure f is closed, automatically